client/my-sites/upgrades/domain-management/controller.jsx

```diff
@@ -58,18 +52,18 @@ module.exports = {
 		renderWithReduxStore(
 			<DomainManagementData
```

Comment:
Looks like `< DomainManagementData />` component isn't updated to work with new property name. There are old references in README file, PropTypes, render method and getStateFromStores funtcion.